Cargo.toml

```diff
@@ -211,22 +211,23 @@ metrics-tracing-context.workspace = true
 
 # AWS - Official SDK
 aws-sdk-s3 = { version = "1.4.0", default-features = false, features = ["behavior-version-latest", "rt-tokio"], optional = true }
-aws-sdk-sqs = { version = "1.3.0", default-features = false, features = ["behavior-version-latest", "rt-tokio"], optional = true }
-aws-sdk-sns = { version = "1.3.0", default-features = false, features = ["behavior-version-latest", "rt-tokio"], optional = true }
-aws-sdk-cloudwatch = { version = "1.3.0", default-features = false, features = ["behavior-version-latest", "rt-tokio"], optional = true }
-aws-sdk-cloudwatchlogs = { version = "1.3.0", default-features = false, features = ["behavior-version-latest", "rt-tokio"], optional = true }
-aws-sdk-elasticsearch = { version = "1.3.0", default-features = false, features = ["behavior-version-latest", "rt-tokio"], optional = true }
-aws-sdk-firehose = { version = "1.3.0", default-features = false, features = ["behavior-version-latest", "rt-tokio"], optional = true }
-aws-sdk-kinesis = { version = "1.3.0", default-features = false, features = ["behavior-version-latest", "rt-tokio"], optional = true }
-aws-sdk-secretsmanager = { version = "1.3.0", default-features = false, features = ["behavior-version-latest", "rt-tokio"], optional = true }
+aws-sdk-sqs = { version = "~1.3.0", default-features = false, features = ["behavior-version-latest", "rt-tokio"], optional = true }
```

Comment:
I'm unsure if we need all of these `~`s rather than just not updating `aws-config` when adding the new crates. Did you find them to be required?

Reply:
Truthfully I was having a hard time running the test locally (was seeing the same result of test failures whether running on tip of master or on this branch) so I may have been a little overzealous here. I'd be happy to try just aws-config if you don't mind triggering the integration test here to check.

Reply:
👍 the integration tests will run during the merge so we should get feedback then. Unfortunately it isn't possible to trigger them on external PRs otherwise at the moment.

Reply:
Gotcha. Okay I'll go ahead and revert the extra version pins and echo back.